omod/src/main/java/org/openmrs/module/owa/filter/OwaFilter.java

```diff
@@ -52,7 +59,15 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
 			requestURL = request.getServletPath();
 		}
 		
-		String loginUrl = Context.getAdministrationService().getGlobalProperty("login.url", "login.htm");
+		String loginUrl;
+		try {

```

Comment:
Isn't the privilege constant available to this module?

Reply:
GET_GLOBAL_PROPERTIES is not available on the platform version this module is using.

Reply:
Can we update the platform version to the very minimum where it is available?

Reply:
I am getting a lot of test errors when updating the platform version.

Reply:
GET_GLOBAL_PROPERTIES is available on 1.11.0+

Reply:
Let us use that.

Reply:
@dkayiwa I created a separate PR for the update: https://github.com/openmrs/openmrs-module-owa/pull/80